data/json/speech.json

```diff
@@ -2262,6 +2262,42 @@
     "sound": "Wanna play with me?",
     "volume": 10
   },
+  {
+    "type": "speech",
+    "speaker": [ "mon_feral_sapien_spear" ],
+    "sound": "Save your tears.",
+    "volume": 10
+  },
+  {
+    "type": "speech",
+    "speaker": [ "mon_feral_sapien_spear" ],
+    "sound": "Soon enough, you'll become one with us, one way or another!",
+    "volume": 15
+  },
+  {
+    "type": "speech",
+    "speaker": [ "mon_feral_sapien_spear" ],
+    "sound": "Be afraid. Be very afraid.",
```

Comment:
```suggestion
    "sound": "Be afraid.  Be very afraid.",
```

Reply:
Thank you, I always forget to look for that...